api/src/scripts/updateDoiNames.ts

```diff
@@ -0,0 +1,165 @@
+import * as client from '../lib/client';
+import * as helpers from '../lib/helpers';
+
+const updateDoiNames = async (): Promise<void> => {
+    const publications = await client.prisma.publication.findMany({
```

Comment:
feels a little like should already be a method to get pubs and there are, but those are in API code, guess this is a 'one shot' script but might set a pattern that is not ideal..

Reply:
there is, as you say, but its an API call and required  some parameters for open search, it was just easier to do this, a bit more "raw" but it is, only a one shot script, I don't think it will set a pattern, specially if we don't keep it around afterwards?

Reply:
PR to delete then 😉